data/human/hails.txt

```diff
@@ -1708,6 +1708,89 @@ phrase "friendly civilian"
 		", but I still had to take a big loan to buy my first ship."
 		", but it was my grandma's inheritance that allowed me to buy this ship."
 
+phrase "friendly civilian"
+	word
+		"Don't be too picky about the jobs you take. It's better to"
+	word
+		" "
+	word
+		"ship crooks, than having a crooked ship."
+		"ship slugs, than having a sluggish ship."
+		"ship bad bands, than having your ship badly banned."
+		"be working hard at shipping, than having your ship hardly working."
+
+
+phrase "friendly civilian"
+	word
+		"Ey Comrade,"
+	word
+		" "
+	word
+		"your ship has loaded on a planet? I have loaded a planet onto my ship."
+		"you have cargo in your ship? I have ships in my cargo."
+		"you have fighters in your bays? We have bays in our fighters."
+
+phrase "friendly civilian"
+	word
+		"Somebody"
+		"Some nut"
+	word
+		" "
+	word
+		"put"
+		"placed"
+	word
+		" "
+	word
+		"a blue phonebooth"
+		"a silver car"
+	word
+		" "
+	word
+		"on my bridge"
+		"in my armoury"
+		"in my cargohold"
+	word
+		" and "
+	word
+		"claimed to be"
+		"told my crew to be"
+	word
+		" "
+	word
+		"a time traveller"
+		"from another time"
+	word
+		", "
+	word
+		"and suddenly it disappeared."
+		"but the only travel he did was to a mental institution."
+		"I still don't know how she got into my spaceship."
+		"the thing looked bigger on the inside than on the outside."
+		"and some weirdos were flying around it on hoverboards."
+
+phrase "friendly civilian"
+	word
+		"I bought "
+	word
+		"a power generator"
+		"explosives"
+		"my new thrusters"
+		"some toothpaste"
+	word
+		" "
+	word
+		"from"
+	word
+		" "
+	word
+		"the ACME company"
+		"the Astronomical Conglomeration of Manufacturing Entities"
+	word
+		", a company many times smaller than the Syndicate."
+		", a minor company but with much better service than the Syndicate."
+		", a pretty insignificant company located just on one planet."
+		" but next time I buy from the much larger Syndicate again."
```

Comment:
I feel like this should be in future tense. Also, I feel like 'the much larger' is unnecessary. It's already almost immediately obvious to a new player that the Syndicate is huge. 
```suggestion
		" but next time I'll buy from the Syndicate again."
```